src/transformers/integrations/ggml.py

```diff
@@ -539,6 +542,12 @@ def tokenizer(self, proto):
                 AddedToken("</s>", normalized=False, special=True),
             ]
         )
+
+        if len(self.proto.added_tokens) != 0:
+            tokenizer.add_special_tokens(
+                [AddedToken(added_token, normalized=False, special=False) for added_token in self.added_tokens]
+            )
```

Comment:
Not all of them are special here. You can add them all as special

Reply:
@younesbelkada this just means that added tokens that are not special will be skipped when decoding.